src/game.cpp

```diff
@@ -10604,6 +10605,24 @@ bool game::walk_move( const tripoint &dest_loc )
         }
     }
 
+    if( u.is_hauling() ) {
+        u.assign_activity( activity_id( "ACT_MOVE_ITEMS" ) );
+        u.activity.values.push_back( false );
+        u.activity.values.push_back( false );
```

Comment:
Duplicate line. Is it intended?

Reply:
Yes, I added comments explaining what's going on.